utils/wasm-gen/src/generator/syscalls/invocator.rs

```diff
@@ -143,6 +146,20 @@ impl<'a, 'b> From<DisabledAdditionalDataInjector<'a, 'b>> for SyscallsInvocator<
 }
 
 impl<'a, 'b> SyscallsInvocator<'a, 'b> {
+    /// Returns the size of the memory in bytes.
+    fn memory_size_bytes(&self) -> u32 {
```

Comment:
I'd move both of methods under all others as "used" ones, not the "users". Or event to a separate `impl`